libmo_unpack/build.sh

```diff
@@ -1,5 +1,8 @@
 #!/bin/bash
 mkdir m4
+
+# Patch the configure.ac as per https://lists.ubuntu.com/archives/fwts-devel/2013-June/003391.html
+sed -i".ac" -e 's/AM_PROG_AR/m4_ifdef([AM_PROG_AR], [AM_PROG_AR])/g' configure.ac
 autoreconf --install
```

Comment:
I was trying to download and install  an updated `autoreconf`.  This is much better!